### DISCUSSION TITLE
Request Payload too large

### DISCUSSION THREAD

**Discussion by sasanknvs** [janhq/jan#3680] [Get Help] [answered]
I am trying to integrate Jan.ai's openai based LLM like gpt-4-turbo and gpt-4o from LaVauge framework to automate the testing. But when tried to integrate it is giving "Request payload too large" error. I am not sure where I can increase the request size limit in Jan.ai. 
When I examined the request that is being sent to the LLM, it has a big image encoded in base64. The whole request content's size is around 2.5MB.

Can anyone please throw some light on this issue and is there any option in Jan.ai where we can increase the request size limit ?

**Comment by louis-jan** [comment, marked as answer]
Hi @sasanknvs could I ask, are you submitting the request through the API or using the chat interface? 

If you are using Chat GUI, please select View > Toggle Developer Tools > Network and duplicating a chat_completion response here? (Please do not attach the request key)

If you are using API, is it possible to compress the image from the client?

![image](https://github.com/user-attachments/assets/73a8a0f9-1706-4f29-89b1-e24ded8cb648)

**Reply by sasanknvs** [reply]
Hi @louis-jan,

I am submitting the request through API. There is a framework called LaVauge where we can integrate with LLM for test automation. I am using their framework and provide the llm context to it and LaVauge does the heavy lifting of integrating with LLM and generating test scripts. 

And I am seeing this issue of payload large only when connected to local running LLM. But when I connect to the remote LLM like chatgpt4 openai, I am not seeing this issue. 

Is there any setting/configuration that can be made so that the local api server accepts larger request payload ?
